docs/rancher/07-run-murmuration-services/migrate-es.md

```diff
@@ -32,7 +28,7 @@ Upon completing this guide, you will have:
 Before starting, ensure you have:
 
 1. Administrative access to both source and destination Kubernetes clusters.
-2. Necessary credentials and access keys for Elasticsearch secure settings.
+2. The access key and secret key for your storage bucket from Amazon S3, DigitalOcean, or another provider.
```

Comment:
I suggest revising it to the following for clearer expression:

The access key and secret key for your storage bucket from Amazon S3, DigitalOcean Space or other S3-compatible cloud services.